### PR TITLE
fix: update protobufjs transitive dependency

### PR DESCRIPTION
**Pull request by schmanu**
## What it solves
Updates `protobufjs`

## How to test it
- Test if trezor wallet still works

## Checklist
* [x] I've tested the branch on mobile 📱
* [x] I've documented how it affects the analytics (if at all) 📊
* [x] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Comment by iamacook** [comment]
For context, this is regarding [this warning](https://github.com/safe-global/safe-wallet-web/security/dependabot/28).

File: package.json [hunk 1]
```diff
@@ -32,6 +32,9 @@
   "pre-commit": [
     "lint"
   ],
+  "resolutions": {
+    "@web3-onboard/trezor/**/protobufjs": "^7.2.4"
+  },
```
[the remark this answers is not in the document]
Comment (schmanu) [reply package.json:37]: It's not possible to add comments in a json file.
I could alternatively create a resolutions.md where we write down why we introduced certain resolutions and when to remove those?

[the remark this answers is not in the document]
Comment (iamacook) [reply package.json:37]: > It's not possible to add comments in a json file.

You can use a workaround:

```json
  "__TODO__": "Remove when @web3-onboard/trezor is updated",
  "resolutions": {
    "@web3-onboard/trezor/**/protobufjs": "^7.2.4"
  },
```

[the remark this answers is not in the document]
Comment (usame-algan) [reply package.json:37]: Oh, I didn't know that. For the sake of simplicity I suggest we leave it as it is then and create a Github issue to keep track of it.

[the remark this answers is not in the document]
Comment (schmanu) [reply package.json:37]: I already subscribed to the dependabot PR which fixes it in the transitive dependency: https://github.com/trezor/trezor-suite/pull/8873

**Review by usame-algan — APPROVED** [review]

**Pull request merged**